frontend/src/app/workflows/workflows.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="btn-toolbar mb-3 mx-2" role="toolbar" aria-label="Toolbar">
+  <div ngbDropdown>
+    <button class="btn btn-primary m-1" id="exampleSelection"
+            ngbDropdownToggle [disabled]="isLoading()">{{currentExampleDesc}}</button>
+    <div ngbDropdownMenu aria-labelledby="exampleSelection">
+      <ng-container *ngFor="let example of examplesList; index as i">
+        <button ngbDropdownItem (click)="changeExampleContent(i)">{{example}}</button>
+      </ng-container>
+    </div>
+  </div>
+
+  <div *ngIf="!isLoading()">
+    <button class="btn btn-primary m-1"
+            (click)="generate()">Generate
+    </button>
+  </div>
+
+  <div *ngIf="isLoading()">
+    <button class="btn btn-primary m-1" type="button" disabled>
+      <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+      Generating...
+    </button>
+  </div>
+
+  <div>
+    <button class="btn btn-success m-1"
+            (click)="download.open()">
+      Download
+    </button>
+  </div>
+</div>
+
+<!-- Main Content Area -->
+<div class="vh-100">
```

Comment:
I think vh-100 is problematic here, because it makes this component slightly larger than the viewport when factoring in the toolbar and the global header. See four-pane-editor, there is a big `<div class="h-100 d-flex flex-column">` around everything and the main content is `<div class="flex-grow-1">`. Probably exactly what you need.